### PR TITLE
Base

### PR DESCRIPTION
**Pull request by Starvosky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    <section>
+      <h3></h3>
+      <p></p>
+    </section>
+    <!-- ^Objectives and Goals -->
+    <section>
+      <h3></h3>
+      <div>
+        <h4></4>
```
Comment (calebgmahala) [inline index.html]: h4

**Review by daviirose — APPROVED** [review]
looks good

**Review by calebgmahala — CHANGES_REQUESTED** [review]
add a favicon
forgot an h4

File: index.html (hunk 1 above)
Comment (Starvosky) [reply index.html]: Alright will fix. Thanks.

**Review by calebgmahala — APPROVED** [review]

**Comment by calebgmahala** [comment]
seal of approval

**Pull request merged**